BackedAkamainCDN/docker-compose.yml

```diff
@@ -2,16 +2,13 @@ version: '3.5'
 
 services:
   akamai_cdn:
-    #network_mode: host
+    network_mode: host
```

Comment:
Zostawiłbym to wykomentowane

Reply:
ta linijka sprawia, że docker nie ma swojej wewnetrznej sieci tyko bierze sieć hosta, potrzebne nam to jest do zczytywania tcpdumpem requestów na maszynie, wcześniej wywaliło, bo zapomniałem usunąć mapowanie portów, które nie jest możliwe bo nie mamy już wewnetrznej sieci, choć co ciekawe, to nie usunąłem, bo jak to dodałem to nie wywalało :D